src/python/ensembl/io/genomio/manifest_stats.py

```diff
@@ -165,36 +171,69 @@ def get_seq_region_stats(self, seq_region_path: Path) -> List[str]:
         # Stats
         stats: List[str] = []
         stats.append(seq_region_path.name)
+        stats += self.coord_systems_stats(coord_systems)
+        stats += self.seq_region_special_stats(circular, locations, codon_tables)
+        stats.append("\n")
+        return stats
+
+    def coord_systems_stats(self, coord_systems: Dict[str, List[int]]) -> List[str]:
+        """For each coord_system compute various stats:
+            - number of sequences
+            - sequence length sum, minimum, maximum, mean
+
+        Args:
+            coord_systems: Coordinate system dictionary of lengths.
+
+        Returns:
+            A list with the computed statistics in a printable format.
+        """
+        stats: List[str] = []
         stats.append(f"Total coord_systems {len(coord_systems)}")
         for coord_name, lengths in coord_systems.items():
             stats.append(f"\nCoord_system: {coord_name}")
 
-            stat_counts: Dict[str, Union[int, float]] = {}
-            stat_counts["Number of sequences"] = len(lengths)
-            stat_counts["Sequence length sum"] = sum(lengths)
-            stat_counts["Sequence length minimum"] = min(lengths)
-            stat_counts["Sequence length mean"] = mean(lengths)
-            stat_counts["Sequence length maximum"] = max(lengths)
+            stat_counts: Dict[str, Union[int, float]] = {
```

Comment:
```suggestion
            stat_counts: Dict[str, float] = {
```
I have just found this on a quick search re. PEP484 (type hints):
> ... when an argument is annotated as having type float, an argument of type int is acceptable; ...